bzkanban.js

```diff
@@ -1062,32 +1079,48 @@ function dropCard(ev) {
     }
 }
 
-function dropBacklog(ev) {
-    if (ev.target.classList.contains("drop-target")) {
-        ev.target.classList.remove("drop-target-hover");
-    }
-    ev.preventDefault();
+function showBacklog() {
+    var button = document.getElementById("btnShowBacklog");
+    var backlogCol = document.querySelector("#BACKLOG.board-column");
+
+    if (backlogCol.style.display === "none" || null) {
```

Comment:
There was a point where we were checking the state of the column but it hadn't been explicitly set to none but was expecting a state that it isn't visible.